turbopack/crates/turbopack-ecmascript/src/references/esm/base.rs

```diff
@@ -43,6 +43,7 @@ pub enum ReferencedAsset {
     Some(Vc<Box<dyn EcmascriptChunkPlaceable>>),
     External(RcStr, ExternalType),
     None,
+    Unresolveable,
```

Comment:
I know you're just moving this code around, but I think we should fix this typo in the process.

```suggestion
    Unresolvable,
```